<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>228</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Rock-paper-scissors</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/img/icon_scissors.png</normaloff>:/img/icon_scissors.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0,0">
    <item>
     <layout class="QHBoxLayout" name="choiceLayout">
      <item>
       <widget class="QRadioButton" name="rockRadioButton">
        <property name="text">
         <string>Rock</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/img/100px-Rock-paper-scissors_(rock).png</normaloff>:/img/100px-Rock-paper-scissors_(rock).png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="paperRadioButton">
        <property name="text">
         <string>Paper</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/img/100px-Rock-paper-scissors_(paper).png</normaloff>:/img/100px-Rock-paper-scissors_(paper).png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scissorsRadioButton">
        <property name="text">
         <string>Scissors</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/img/100px-Rock-paper-scissors_(scissors).png</normaloff>:/img/100px-Rock-paper-scissors_(scissors).png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="randomPushButton">
        <property name="text">
         <string>Random</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="opponentHashLabel">
        <property name="text">
         <string>Enter what your opponent sent:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="myCleartextLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="opponentCleartextLabel">
        <property name="text">
         <string>Enter what your opponent sent:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="opponentCleartextLineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="myHashLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>380</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="opponentHashLineEdit"/>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="myHashLabel">
          <property name="text">
           <string>Send this:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="copyMyHashButton">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="myCleartextLayout">
        <item>
         <widget class="QLabel" name="myCleartextLabel">
          <property name="text">
           <string>Send this:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="myCleartextSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="copyMyCleartextButton">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="newGameButton">
      <property name="text">
       <string>New game</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="donateLabel">
      <property name="toolTip">
       <string>Contributions let me work at things like this
instead of earning money IRL.
More contributions =&gt; more funny applications</string>
      </property>
      <property name="text">
       <string>Your contribution to address 1N8kkCSKFc18LEtBLLSdoa35cdxsLRvLXM is highly appreciated</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
